chess_gui.py

```diff
@@ -85,23 +90,74 @@ def highlight_square(screen, game_state, valid_moves, square_selected):
                 screen.blit(s, (move[1] * SQ_SIZE, move[0] * SQ_SIZE))
 
 
+def game_state_log(game_state, white_turns_with_all_pieces, black_turns_with_all_pieces):
+    pieces_white = 0
+    pieces_black = 0
+    for r in range(DIMENSION):
+        for c in range(DIMENSION):
```

Comment:
Rename 'r' and 'c'   
Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing